agent/util-scripts/pbench-tool-meister

```diff
@@ -81,35 +82,44 @@ class ToolMetadata:
             self.json = Path(install_path, "tool-scripts", "meta.json")
         self.data = None
 
-    #TODO: ADD BETTER ERROR CHECKING TO THIS METHOD
     def getFullData(self):
         if self.data:
             return self.data
 
         if self.mode == "json":
+            if not os.path.isfile(self.json):
+                self.logger.exception('There is no tool-scripts/meta.json in given install dir')
```

Comment:
`logger.exception` wants to be called from an exception handler context; since this isn't, it should really be `logger.error` instead.